.bazelrc

```diff
@@ -16,6 +16,10 @@ build --noenable_bzlmod
 # This is a mandatory flag.
 build --incompatible_default_to_explicit_init_py
 
+# Don't let local Python site packages leak into the build and cause problems
+common --incompatible_strict_action_env
```

Comment:
This one is already in this file, perhaps change that one from build to common.

Reply:
changed